pytket/extensions/cutensornet/structured_state/mps_gate.py

```diff
@@ -171,54 +172,32 @@ def _apply_2q_unitary(self, unitary: cp.ndarray, q0: Qubit, q1: Qubit) -> MPSxGa
                 normalization="L2",  # Sum of squares singular values must equal 1
             )
 
-        else:
-            # The user did not explicitly ask for truncation, but it is advantageous to
-            # remove any singular values below ``self._cfg.zero``.
-            self._logger.debug(f"Truncating singular values below={self._cfg.zero}.")
-            if self._cfg.zero > self._cfg._atol / 1000:
-                self._logger.info(  # This was raised as a warning in Config already
-                    "Your chosen value_of_zero is relatively large. "
-                    "Faithfulness of final fidelity estimate is not guaranteed."
-                )
-
-            # NOTE: There is no guarantee of canonical form in this case. This is fine
-            # since canonicalisation is just meant to detect the optimal singular values
-            # to truncate, but if we find values that are essentially zero, we are safe
-            # to remove them.
-            svd_method = tensor.SVDMethod(
-                abs_cutoff=self._cfg.zero,
-                partition="U",  # Contract S directly into U (named L in our case)
-                normalization=None,  # Without canonicalisation we must not normalise
-            )

```

Comment:
Combining with the case of truncation for a given `max_extent` (`chi`) since there's no longer need to make a distinction now that we always canonicalise, no matter whether or not the user requested truncation.